extractors/personal_identifiers/location_extraction/code_snippet_common.md

```diff
@@ -0,0 +1,35 @@
+```python
+import spacy
+from typing import List, Tuple
+
+def location_extraction(text: str, label: str) -> List[Tuple[str, int]]:
+    """
+    @param text: the input text
+    @param label: the label that is assigned to extracted words
+    @return: positions of extracted names of persons  
+    """
+    nlp = spacy.load("en_core_web_sm")
```

Comment:
same as last time, always loading can result in performance issues so a multiton should do the trick

Reply:
Moved out outside of the function now.